frontend/src/components/v2/SecretInput/SecretInput.tsx

```diff
@@ -36,7 +59,7 @@ const syntaxHighlight = (orgContent?: string | null, isVisible?: boolean) => {
       `<span class="ph-no-capture text-yellow">&#36;&#123;<span class="ph-no-capture text-yello-200/80">${b}</span>&#125;</span>`
```

Comment:
@akhilmhdh Also can you explain what this line of code does? I'd like to know because I removed the return type of this function.

Reply:
Good to know. I can replace it back with not removing the feature.

Reply:
And btw the dots are not calculated from the starting of `<` or `>`.

If the secret is `Trigger.dev <hello@trigger>` it has only 12 dots whereas it should have 27. I know where it's been calculated. It's happening because of the check `!isHtml && char !== "\n"` from `replaceContentWithDot`

Reply:
All I need to know is whether the dots should replace all the letters inside and including the angle brackets. Or is it better left like now?

Reply:
@akhilmhdh Other than the above comment. Anything else needs to be checked?

Reply:
Let me check this and ill get back to you